### ISSUE TITLE
alephone fails to start the Marathon Rubicon X fan game

### ISSUE THREAD

**Issue by orbea** [Aleph-One-Marathon/alephone#95]
OS: `Slackware64-current`

Game - http://files5.bungie.org/marathon/marathonRubiconX.zip
md5sum - a91a091d4b02082464e7a93f551ba221

When using the alephone github master (b052012) with the `Marathon Rubicon X` fan game it will fail to start with the following garbled error message.
```
fatal alert (ID=-1): Please be sure the files ubicon Map ubicon Shapes ubicon Imagesand "Rubicon Soundsare in the same folder as the ubiconapplication and try again. (csalerts_sdl.cpp:210)
```
Full log.
```
$ alephone /tmp/rubicon_x/
Aleph One Linux 2017-02-05 1.3a2
https://alephone.lhowon.org/

Original code by Bungie Software <http://www.bungie.com/>
Additional work by Loren Petrich, Chris Pruett, Rhys Hill et al.
TCP/IP networking by Woody Zenfell
Expat XML library by James Clark
SDL port by Christian Bauer <Christian.Bauer@uni-mainz.de>

This is free software with ABSOLUTELY NO WARRANTY.
You are welcome to redistribute it under certain conditions.
For details, see the file COPYING.

Built with network play enabled.

Built with Lua scripting enabled.
loop_start=1 and loop_end=57900 but bytes_per_frame=2; interpreting as frame offsets (SoundFile.cpp:111)
GL_VENDOR: nouveau
GL_RENDERER: NVF1
GL_VERSION: 3.1 Mesa 18.1.0-devel (git-776e6af879)
fatal alert (ID=-1): Please be sure the files ubicon Map ubicon Shapes ubicon Imagesand "Rubicon Soundsare in the same folder as the ubiconapplication and try again. (csalerts_sdl.cpp:210)
```
Contents of the game directory.
https://pastebin.com/yjugDJTq

I see two issues here.
1. The error message is garbled including the graphical error prompt. What is it supposed to say?
2. These fan games may never be updated so it would be good to not break compatibility. At the very least it would be nice to know how to fix the game so I can fix it for slackbuilds.org if and when alephone is updated there.

**Comment by Hopper262** [comment]
Rubicon X should work now in the latest Aleph One.

1. The error message is garbled due to an error in the Rubicon X scenario files (the message text was written by the Rubicon devs and has always been garbled AFAICT). The message wouldn't have helped, because an unrelated bug caused this error to show up incorrectly.

2. The error was due to a bug fixed in f0617de - Rubicon X should launch correctly under current master or the latest beta (1.3b2).

**Comment by orbea** [comment]
Thanks, I can confirm that marathon-rubicon does start now.
